bundles/action/org.openhab.action.pushbullet/README.md

```diff
@@ -4,12 +4,19 @@ The Pushbullet action allows you to notify iOS, Android & Windows 10 Phone & Des
 
 ## Actions
 
-The following is a valid action call that can be made when the plugin is loaded. For specific information on each item, see the [Pushbullet API](https://docs.pushbullet.com/).
+The following is a valid action call that can be made when the plugin is loaded.
+For specific information on each item, see the [Pushbullet API](https://docs.pushbullet.com/).
+
+* `sendPushbulletNote(String receiver, String title, String message)`
+* `sendPushbulletNote(String botname, String receiver, String title, String message)`
+
+## Examples
 
 ```
-pushbullet(String botname, String receiver, String title, String message)
+sendPushbulletNote("pitaya", "hakan.tandogan@gmail.com", "this is the title", "And this is the body of the message")

```

Comment:
Are you sure you want your private email exposed in the docs? :-)

Reply:
aaaah, now you mention it :)

Reply:
And removed, thank you for seeing this.